docs/source/guide/session.rst

```diff
@@ -62,9 +65,9 @@ You can configure each session with specific credentials, AWS Region information
 Multithreading or multiprocessing with sessions
 -----------------------------------------------
 
-Similar to ``Resource`` objects, ``Session`` objects are not thread safe
-and should not be shared across threads and processes. It's recommended
-to create a new ``Session`` object for each thread or process::
+Similar to ``Resource`` objects, ``Session`` objects are not thread safe and should not be shared across threads and processes. You should create a new ``Session`` object for each thread or process:
```

Comment:
We should probably line wrap these like the rest of the guides.

Reply:
Yeah, I've had trouble keeping these consistent because the existing content is so inconsistent about it. Some stuff just relies on auto-wrapping while other places have hard wraps, but widths aren't even consistent, varying anywhere from about 76 characters to something closer to 110 characters.